docs/sources/operator/api.md

```diff
@@ -0,0 +1,4618 @@
+---
+aliases:
+- /docs/agent/latest/operator/crd/
+title: CRD Reference
+weight: 500
+---
+# CRD Reference
+<p>Packages:</p>
+<ul>
+<li>
+<a href="#monitoring.grafana.com%2fv1alpha1">monitoring.grafana.com/v1alpha1</a>
+</li>
+</ul>
+<h2 id="monitoring.grafana.com/v1alpha1">monitoring.grafana.com/v1alpha1</h2>
+Resource Types:
+<ul><li>
+<a href="#monitoring.grafana.com/v1alpha1.Deployment">Deployment</a>
+</li><li>
+<a href="#monitoring.grafana.com/v1alpha1.GrafanaAgent">GrafanaAgent</a>
+</li><li>
+<a href="#monitoring.grafana.com/v1alpha1.IntegrationsDeployment">IntegrationsDeployment</a>
+</li><li>
+<a href="#monitoring.grafana.com/v1alpha1.LogsDeployment">LogsDeployment</a>
+</li><li>
+<a href="#monitoring.grafana.com/v1alpha1.MetricsDeployment">MetricsDeployment</a>
+</li></ul>
+<h3 id="monitoring.grafana.com/v1alpha1.Deployment">Deployment
+</h3>
+<div>
+<p>Deployment is a set of discovered resources relative to a GrafanaAgent. The
+tree of resources contained in a Deployment form the resource hierarchy used
+for reconciling a GrafanaAgent.</p>
+</div>
+<table>
+<thead>
+<tr>
+<th>Field</th>
+<th>Description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td>
+<code>apiVersion</code><br/>
+string</td>
+<td>
+<code>
+monitoring.grafana.com/v1alpha1
+</code>
+</td>
+</tr>
+<tr>
+<td>
+<code>kind</code><br/>
+string
+</td>
+<td><code>Deployment</code></td>
+</tr>
+<tr>
+<td>
+<code>Agent</code><br/>
+<em>
+<a href="#monitoring.grafana.com/v1alpha1.GrafanaAgent">
+GrafanaAgent
+</a>
+</em>
+</td>
+<td>
+<p>Root resource in the deployment.</p>
+</td>
+</tr>
+<tr>
+<td>
+<code>Metrics</code><br/>
+<em>
+<a href="#monitoring.grafana.com/v1alpha1.MetricsDeployment">
+[]MetricsDeployment
```

Comment:
Should these brackets be here?  There are other places where they show up, too.

Reply:
This is automatically generated by the tool meaning that this section is composed by a list of `MetricsDeployment`